security/pkg/nodeagent/sds/server.go

```diff
@@ -167,6 +167,10 @@ func (s *Server) initWorkloadSdsService(options *Options) error {
 			if err = s.grpcWorkloadServer.Serve(s.grpcWorkloadListener); err != nil {
 				log.Errorf("SDS grpc server for workload proxies failed to start: %v", err)
 			}
+			s.grpcWorkloadListener, err = setUpUds(options.WorkloadUDSPath)
+			if err != nil {
+				log.Errorf("SDS grpc server for workload proxies failed to set up UDS: %v", err)
```

Comment:
Please give a different error message for debugging purpose.

Reply:
"workload proxies failed to set up UDS" is different from "workload proxies failed to start".